client/verta/tests/test_cli/test_registry.py

```diff
@@ -717,3 +717,53 @@ def test_creation(self, client, created_entities, data_type_cls):
         created_entities.append(registered_model)
 
         assert registered_model.get_data_type() == data_type
+
+
+class TestModelIODescription:
+   def test_create_version(self, client, created_entities):
+        LogisticRegression = pytest.importorskip('sklearn.linear_model').LogisticRegression
+        model_name = RegisteredModel._generate_default_name()
+        version_name = "my version"
+        input_description = "my input description"
+        output_description = "my ouput description"
+
+        filename = "tiny1.bin"
+        FILE_CONTENTS = os.urandom(2**16)
+        with open(filename, 'wb') as f:
+            f.write(FILE_CONTENTS)
+
+        classifier_name = "tiny2.pth"
+        classifier = LogisticRegression()
+        with open(classifier_name, 'wb') as f:
+            pickle.dump(classifier, f)
+
+        runner = CliRunner()
+        result = runner.invoke(
+            cli,
+            ['registry', 'create', 'registeredmodel', model_name, '-l', 'label1', '-l', 'label2'],
```

Comment:
We can skip these labels since we're not testing them.

```suggestion
            ['registry', 'create', 'registeredmodel', model_name],
```